sale_order_product_recommendation/wizards/sale_order_recommendation_view.xml

```diff
@@ -28,6 +29,38 @@
                                 <field name="units_delivered"/>
                                 <field name="units_included" widget="numeric_step"/>
                             </tree>
+                            <kanban quick_create="0" create="0" delete="0">
+                                <field name="id"/>
+                                <field name="product_id"/>
+                                <field name="price_unit"/>
+                                <field name="sale_uom"/>
+                                <field name="times_delivered"/>
+                                <field name="units_delivered"/>
+                                <field name="units_included"/>
+                                <field name="currency_id"/>
+                                <templates>
+                                    <t t-name="kanban-box">
+                                        <div class="oe_kanban_global_click">
+                                            <div class="o_kanban_image">
+                                                <img t-att-src="kanban_image('product.template', 'image_small', record.product_id.raw_value)" alt="Product"/>
+                                            </div>
+                                            <div class="oe_kanban_details">
+                                                <div><field name="product_id"/></div>
+                                                <div>Price: <field name="price_unit" widget="monetary" options="{'currency_field': 'currency_id', 'field_digits': True, 'style': 'margin-bottom: 0px'}" class="mt-0"/></div>
```

Comment:
```suggestion
                                                <div>Price: <field name="price_unit" widget="monetary" options="{'currency_field': 'currency_id', 'field_digits': True}" class="mt-0 mb-0"/></div>
```

Reply:
This was several tests @sergio-teruel and me did. Do you think with that it's going to work?

Reply:
I think it should work, but if you tested it and it doesn't, this is also not a critical thing... It's just a good rule of thumb to use classes instead of inline styles wherever possible.